rebar.config

```diff
@@ -35,6 +35,10 @@
   deprecated_functions
  ]}.
 
+{relx,
+  [{sys_config, "config/sys.config"}
```

Comment:
this has been a lib up until now and not a release, why does that need to change?